watertap/costing/unit_models/heater_chiller.py

```diff
@@ -140,6 +140,13 @@ def cost_chiller(blk, cost_electricity_flow=True):
             be converted to kW and costed as an electricity. Defaults to True.
     """
     t0 = blk.flowsheet().time.first()
+    blk.effective_heat_duty = pyo.Var(
+        domain=pyo.NonNegativeReals, initialize=0, units=pyo.units.watt
```

Comment:
I think this should also be indexed by time. Also should add doc string.

Reply:
addressed